test/unit/ngsiv2/examples/contextRequests/updateContextAliasPlugin2.json

```diff
@@ -1,6 +1,10 @@
 {
   "luminance": {
-    "type": "lumens",
-    "value": "9"
+    "type": "Number",
+    "value": 9,
+    "metadata": {
+		"type":"Property",
```

Comment:
Indent style seems to be broken in this file in the `metadata` field... Please check.

Reply:
Fixed 7ba9d12